content/en/security_platform/cspm/findings.md

```diff
@@ -46,33 +46,37 @@ A finding is the primary primitive for a rule evaluation against a resource. Eve
+In order to consume findings in logical groupings and mitigate the potential for alert fatigue, you have full flexibility to change how signals are triggered for each individual resource every time a resource fails a rule in a new cloud account each time a resource is misconfigured in a service or to trigger off any Datadog facet. Regardless of which grouping logic you choose for signal generation, opening a signal always displays the up-to-date list of findings that are failing for this rule.
 
 Clicking on an individual finding that has **failed** to see details about the misconfigured resource, the rule description, its framework or industry benchmark mapping, and suggested remediation steps.
 
 {{< img src="security_platform/cspm/findings/signal.png" alt="An example of a signal" style="width:80%;">}}
 
-From the top of the side panel, you can configure a rule, or send a notification to your colleagues by email, Slack, Microsoft Teams, Pagerduty, Servicenow, Jira, and webhooks.
+From the top of this panel, you can configure a rule, or send a notification to your colleagues by email, Slack, Microsoft Teams, Pagerduty, Servicenow, Jira, webhooks, and more.
+
+{{< img src="security_platform/cspm/findings/side-panel-top.png" alt="View related findings and set a rule from the top of the side panel" style="width:80%;">}}
 
 The top portion of the **Findings** side panel also displays key information about where misconfigurations are occurring: an individual resource, a service, or a cloud account. See all tags associated with this grouping so you can quickly find this area within your infrastructure for deeper inspection. It also highlights the relevant compliance controls associated with the rule, and when the signal was first triggered by a finding.
 
+{{< img src="security_platform/cspm/findings/findings.png" alt="View individual resources, a service, or cloud account from the top of the side panel" style="width:80%;">}}
+
 Below is example message for a rule, including a description of misconfiguration and instructions for how to remediate the issue.
 
 {{< img src="security_platform/cspm/findings/message.png" alt="Message tab of the signals panel" style="width:80%;">}}
 
-The Findings tab displays all `evaluation:fail` resources that are triggering this signal.
+The Findings tab displays all findings that are triggering this signal. This list always shows the current state of your infrastructure, meaning if you fixed three of ten misconfigured security groups since the signal first triggered, Datadog will display seven failed security groups rather than display findings that no longer are in violation.
```

Comment:
```suggestion
The Findings tab displays all findings that are triggering this signal. This list always shows the current state of your infrastructure, meaning if you fixed three of ten misconfigured security groups since the signal first triggered, Datadog displays seven failed security groups rather than displaying findings that no longer are in violation.
```